test/models/concerns/audit_extensions_test.rb

```diff
@@ -133,4 +133,49 @@ def setup
       assert_equal [@org.id], audit.organization_ids
     end
   end
+
+  describe '.untaxed and .taxed_and_untaxed' do
+    setup do
+      @new_org = taxonomies(:organization2)
+      @old_org = Organization.current
+      Organization.current = @new_org
+    end
+
+    describe '.untaxed' do
+      subject { Audit.untaxed }
+
+      test 'returns audits only for untaxed resources' do
+        taxed, untaxed = prepare_taxed_and_untaxed_audits_as(org: @new_org)
+        assert subject.reload.include?(untaxed.audits.last)
+        assert_not subject.reload.include?(taxed.audits.last)
+      end
+    end
+
+    describe '.taxed_and_untaxed' do
+      subject { Audit.taxed_and_untaxed }
+
+      test 'returns audits for untaxed and resources' do
+        taxed, untaxed = prepare_taxed_and_untaxed_audits_as(org: @new_org)
+        assert subject.reload.include?(untaxed.audits.last)
+        assert subject.reload.include?(taxed.audits.last)
+      end
+    end
+
+    teardown do
+      Organization.current = @old_org
+    end
+  end
+
+  private
+
+  def prepare_taxed_and_untaxed_audits_as(org: nil, loc: nil)
+    taxed, untaxed = nil
+    Taxonomy.as_taxonomy(org, nil) do
+      as_admin do
+        taxed = FactoryBot.create(:host, :with_auditing, :location => loc, :organization => org)
```

Comment:
Could we please add one more regularly taxed resource? Domain could be a good example. Also it would be good to have one org xor loc only, Organization would be a good representant.